.travis.yml

```diff
@@ -1,13 +1,19 @@
-language: python
+# This is a Tracis CI configuration file
```

Comment:
It's great that you've added comments for these lines! Could you make a separate PR just with this travis.yaml? I would like to merge this to master.